content/docs/setup/kubernetes/getting-started/index.md

```diff
@@ -1,41 +1,42 @@
 ---
 title: Getting Started
-description: Download and install Istio.
+description: Download, install, and try out Istio.
 weight: 5
 skip_toc: true
-keywords: [kubernetes]
+keywords: [getting-started, install, bookinfo, quick-start, kubernetes]
 ---
 
-If you’re new to Istio and just want to try it out, the quickest way to get started is
-by downloading and installing Istio's built-in **demo** configuration.
-Istio can also be installed and customized in many other ways, depending on your platform and intended use.
-Check out our [general installation instructions](/docs/setup/) for details.
-
-To get started, proceed as follows:
+This guide lets you try out Istio quickly and it's the ideal starting
+point if you are new to the project. To get started, download and install
+Istio's built-in **demo** configuration:
 
 1. [Setup your platform](/docs/setup/kubernetes/platform-setup/)
 1. [Download the Istio release](/docs/setup/kubernetes/#downloading-the-release)
 1. [Follow the quick-start installation instructions](/docs/setup/kubernetes/install/kubernetes)
 
-Once you've installed Istio, a good way to start evaluating Istio's features is using our
-[Bookinfo sample application](/docs/examples/bookinfo/).
-You can use this sample to experiment with Istio’s features for traffic routing,
-fault injection, rate limiting, etc..
+With Istio installed, you can [deploy the Bookinfo sample](/docs/examples/bookinfo/#if-you-are-running-on-kubernetes)
```

Comment:
This is pointing into the middle of the bookinfo sample guide (the deploy it on kubernetes part), which isn't idea, since they have no idea what it is at this point. Previously I pointed to the top of the document, so people can read what the sample does first. Then I had a second link to the how to deploy part, so they can deploy it before they start the tasks that depend on it.

Reply:
Readers of quick starts expect to be able to jump right into testing things. By linking directly to the instructions, we satisfy that expectation. If readers are curious, they will instinctively scroll up to find out more.